test/__tests__/lambda.test.js

```diff
@@ -40,6 +41,7 @@ describe(`${name} Tests`, () => {
   describe("Router Tests", () => {
     it("Payload", async () => {
       const routerOutput = await transformer.processRouterDest(inputRouterData);
+      assertRouterOutput(routerOutput, inputRouterData);
```

Comment:
@ItsSudip why are these files still present? didn't we move them to component testing?

Reply:
Rebased this pr onto develop after Sudip's PR is merged.